chainsync/chainsync.go

```diff
@@ -4,9 +4,15 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/filecoin-project/lotus/api/apistruct"
 	"github.com/filecoin-project/lotus/chain/types"
-	"github.com/textileio/lotus-client/api/apistruct"
-	"github.com/textileio/lotus-client/chain/store"
+)
+
+const (
+	hcApply = "apply"
+	// For completeness:
+	// hcRevert = "revert"
+	// hcCurrent = "current"
 )
```

Comment:
Now `lotus/api` is CGO free. But of course not all other subpackages. The `chain/store` subpackage isn't CGO free and here was used to access some constants. I moved those constants here to avoid a single constants use break all PG with CGO stuff.